### PR TITLE
docs: Image Pull Secret Doc

### PR DESCRIPTION
**Pull request by SNe789**
# Description

- Added a new section for Auto Image Pull Secret.

### CODE REVIEW

File: docs/user-guide/global-configurations/docker-registries.md [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
 
-![](../../user-guide/global-configurations/images/Container_Registry.jpg)
 
 ### Name
 
-Provide a name to your registry, this name will be shown to you in Docker Build Config as a drop-down.
+Provide a name to your registry and this name will be shown to you in Build Configuration in the drop-down list.
 
-### Registry type
+### Registry Type
```
Comment (pghildiyal) [inline docs/user-guide/global-configurations/docker-registries.md:45]: its should be ECR

File: docs/user-guide/global-configurations/docker-registries.md [hunk 2]
```diff
@@ -1,37 +1,48 @@
 # Container Registries
 
-Container registries are used to store images built by the CI Pipeline. Here you can configure the container registry you want to use for storing images.
+Container registries are used for storing images built by the CI Pipeline. You can configure the container registry using any different cloud provider of your choice. It allows you to build, deploy and manage your container images with easy-to-use UI.
```
Comment (pghildiyal) [inline docs/user-guide/global-configurations/docker-registries.md:3]: its not cloud provide its container registry of your choice

File: docs/user-guide/global-configurations/docker-registries.md (hunk 1 above)
Comment (SNe789) [reply docs/user-guide/global-configurations/docker-registries.md]: This is a common "Registry Type", it was defined again in detail. It's confusing so I have removed it as it is already mentioned in the table while adding container registry. Now, I have included clickable link for each registry type to keep the navigation easier.

File: docs/user-guide/global-configurations/docker-registries.md [hunk 3]
```diff
@@ -1,43 +1,52 @@
 # Container Registries
 
-Container registries are used to store images built by the CI Pipeline. Here you can configure the container registry you want to use for storing images.
+Container registries are used for storing images built by the CI Pipeline. You can configure the container registry using any different container registry provider of your choice. It allows you to build, deploy and manage your container images with easy-to-use UI.
```
Comment (prakarsh-dt) [inline docs/user-guide/global-configurations/docker-registries.md:3]: `using any different` sounds repetitive.

Comment (SNe789) [reply docs/user-guide/global-configurations/docker-registries.md]: Removed "different".

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**